### PR TITLE
fix(GRO-371): Updates popovers; prevents artist suggestion popover from overflowing

### PR DESCRIPTION
**Pull request by dzucconi**
Closes: [GRO-371](https://artsyproduct.atlassian.net/browse/GRO-371)

I'm impressed with how much of a rabbit hole I managed to turn this into. 😅 

I left the unstated stuff alone for now though I'd like to follow up with it's removal.

### CODE REVIEW

File: src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx [hunk 1]
```diff
@@ -129,23 +128,17 @@ export class ArtworkActions extends React.Component<ArtworkActionsProps> {
 
   renderShareButton() {
     return (
-      <MediocrePopover
```
Comment (damassi) [inline src/v2/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx:132]: lol

File: src/v2/Components/FollowButton/FollowArtistButton.tsx [hunk 2]
```diff
@@ -169,66 +157,69 @@ export class FollowArtistButton extends React.Component<Props, State> {
...
+              artist={artist!}
             />
-          </SuggestionsPopoverContainer>
-        )}
+          }
+        >
+          {({ anchorRef, onVisible }) => {
+            const openSuggestions = () => {
+              if (triggerSuggestions && !artist?.is_followed) {
+                onVisible()
```
Comment (damassi) [inline src/v2/Components/FollowButton/FollowArtistButton.tsx:186]: what does `onVisible` do?

**Review by damassi — APPROVED** [review]
(Thanks for the additional `STRICT_NULL_CHECK` cleanup)

File: src/v2/Components/FollowButton/FollowArtistButton.tsx (hunk 2 above)
Comment (dzucconi) [reply src/v2/Components/FollowButton/FollowArtistButton.tsx:186]: Opens the popover

Comment (damassi) [reply src/v2/Components/FollowButton/FollowArtistButton.tsx:186]: I see, so you can set `<Popover visible /> or use this form to display 👍

Comment (dzucconi) [reply src/v2/Components/FollowButton/FollowArtistButton.tsx:186]: Exactly

**Pull request merged**